src/Sarif.Multitool.FunctionalTests/TestData/DeserializationError_Expected.sarif

```diff
@@ -83,7 +83,7 @@
               "message": {
                 "text": "Analyzing 'DeserializationError.sarif'..."
               },
-              "level": "note",
+              "level": "none",
```

Comment:
>none [](start = 24, length = 4)

These are trace messages and in no way a problem report, so "none" is the correct level.